javaparser-core/src/main/java/com/github/javaparser/ast/Node.java

```diff
@@ -575,7 +577,7 @@ public void register(AstObserver observer, ObserverRegistrationMode mode) {
         if (mode == null) {
             throw new IllegalArgumentException("Mode should be not null");
         }
-        switch (mode) {
+        switch(mode) {
```

Comment:
space or not space? This is the question :D

Reply:
Code generator :-)